packages/db/src/search/CollectionSearch.ts

```diff
@@ -114,22 +134,20 @@ export class CollectionSearch extends BaseSearch {
   private async retrieveGranuleStats(collectionCumulusIds: number[], knex: Knex)
     : Promise<StatsRecords> {
     const granulesTable = TableNames.granules;
-    const statsQuery = knex(granulesTable)
```

Comment:
when granuleQuery is being cloned, it isn't then being used anywhere else (although statsQuery is) can this be simplified to use granuleQuery or statusQuery variable the whole way through? I found this very confusing to read

Reply:
Good comment.  Updated [56dd619](https://github.com/nasa/cumulus/pull/3861/commits/56dd619cb1d9abb8829d9f9dbe65cc2c8cc972c3)

Reply:
yea this looks much more understandable to me, thanks